packages/mirage-runtime/mirage-runtime.3.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://mirage.io/releases/mirage-3.0.5.tbz"
```

Comment:
Ah! This is the output of the `make opam-pkg` step. Not quite as perfect as I thought...